src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -24,15 +24,15 @@
         <div v-else class="summary-desc">
           <div><v-icon color="green darken-2" class="agreement-valid-icon">mdi-check</v-icon></div>
             <div>
-              {{ selectedAgreementDescription }}
+              {{ getIncorporationAgreement.description }}
```

Comment:
OK, I see you're reusing this component for the ULC/LTD/CCC companies...

We need to be careful how these components are organized (folder-wise) in this app. They're currently in folders according to step name, but those steps will change for different companies...
1. I think we should organize these components by component name instead (right now they might match so no change required?)
2. I think we should flatten unnecessary folders, for example any step folder with only 1 component in it

I'm not exactly sure what future changes we will need to make... I am trying to avoid painting ourselves into a corner where a component is in a folder that doesn't make sense.

Reply:
(Also need to ensure state isn't organized by step number. Maybe it already isn't -- I haven't checked -- busy with 1M things atm, sorry.)

Reply:
For sure, i don't disagree here. This seems like it falls into the realm of the store refactor that we discussed earlier though doesn't it? 
- Refactor the store and component directories to be separate of the steps, would fall into the framework no?

The current iteration of these resources are separate of the steps within the store 👍